### ISSUE TITLE
Add term abnormal ossification of ischium

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10417]
**Preferred term label:**
Abnormal ossification of the ischium

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormality of the formation and mineralization of ischial bones. PMID: 9933334

**Parent term (use hpo.jax.org/app)**
Abnormal ischium morphology HP:0003174

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Achondrogenesis 1A (omim200600) PMID: 23956106
Cleidocranial dysplasia (omim119600) PMID: 9933334

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000654
- label: Abnormal ischium ossification
- parents: [Abnormal ischium morphology (HP:0003174)](https://hpo.jax.org/app/browse/term/HP:0003174)
- pmid: PMID:9933334
- def: An abnormality of the formation and mineralization of ischial bones.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10417
- synonyms: Abnormal ossification of the ischium(NONE)
